modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -69,3 +69,24 @@ $ oc apply -f max-worker-pods.yaml
 ----
 $ oc get kubeletconfigs set-max-pods -o yaml
 ----
+
+By default, only 1 machine is allowed to be unavailable when applying the kubelet-related config to the available worker nodes. This will take a lot of time for the config change to get reflected for a large cluster. The number of machines that can be updating at any given time can be tweaked for speeding up the process.
+
+.procedure
+.Run:
++
+----
+$ oc edit machineconfigpool worker
+----
++
+.Set `maxUnavailable` to the desired value.
++
+----
+spec:
+  maxUnavailable: <node_count>
+----
+
+[IMPORTANT]
+====
+Setting the `maxUnavailable` to a large value can lead to application downtime. The value needs to be set taking into consideration the number of worker nodes that can be unavailable without affecting the applications running on the cluster.
```

Comment:
When setting the value, consider the number of worker nodes that can be unavailable without affecting the applications running on the cluster.